docusaurus.config.js

```diff
@@ -52,43 +52,69 @@ const config = {
       '@docusaurus/plugin-client-redirects',
       {
         redirects: [
-          //Latest Links
-          {
-            to: '/docs/participate/contribute',
-            from: '/docs/community/contribute/',
-          },
-          {
-            to: '/docs/develop/intro',
-            from: ['/docs/category/developer-documentation/','/docs/developers/intro/', '/docs/pre-release/category/develop-on-nova-evm/']
-          },
-          {
-            to: '/docs/farming-&-staking/staking/intro',
-            from: ['/docs/operators_and_staking/intro', '/docs/category/operators-and-nominators']
-          },
-          {
-            to: '/docs/farming-&-staking/farming/advanced-cli/cli-install',
-            from: ['/docs/farming-&amp;-staking/farming/advanced-cli/cli-install', '/docs/protocol/cli/', '/docs/protocol/substrate-cli/', '/docs/Farming%20&%20Staking/Farming/Advanced-Cli/cli-install', '/docs/category/advanced-cli-recommended', '/docs/category/advanced-cli']
-          },
-          {
-            to: '/docs/farming-&-staking/timekeeping',
-            from: '/docs/protocol/timekeeping'
-          },
-          {
-            to: '/docs/category/farming',
-            from: ['/docs/category/get-started-with-farming', '/protocol/farm/farming']
-          },
-          {
-            to: '/docs/farming-&-staking/farming/space-acres/install',
-            from: ['/docs/protocol/pulsar/', '/docs/category/pulsar-recommended', '/docs/category/pulsar', '/docs/category/space-acres', '/docs/farming-&-staking/farming/pulsar', '/docs/category/space-acres-recommended', '/docs/farming-&-staking/farming/space-acres/space-acres-install', '/docs/farming-&amp;-staking/farming/space-acres/space-acres-install', '/docs/farming-&amp;-staking/farming/space-acres/install']
-          },
-          {
-            to: '/docs/farming-&-staking/staking/operators/register-operator',
-            from: '/docs/farming-&-staking/staking/operators'
-          },
-          {
-            to: '/docs/farming-&-staking/farming/intro/',
-            from: '/docs/farming-&-staking/farming/prerequisites/',
-          },
+          // Redirect Links
```

Comment:
These looks correct at a glance, will double check before merging :)

Reply:
I built all paths from the slugs first and then added all existing redirects, so they should be correct. I made one more change: in the original redirects, `'/docs/pre-release/category/develop-on-nova-evm/'` was listed as a redirect. It seems odd that a pre-release URL would be shared, so I changed it to `'/docs/category/develop-on-nova-evm/'`.